osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/infra/networking/DlmsPushNotificationDecoder.java

```diff
@@ -66,26 +86,30 @@ protected void decode(
      * 37 38 39 30 31 32 // Equipment id EXXXX123456789012 09 06 00 01 19 09 00 FF // Logical name:
      * Push setup alarms 06 00 00 00 02 // Alarm register, with Replace battery set
      *
-     * <p>Notes: - For SMR5 alarms, we get 8 additional addressing bytes in front of the alarm.
-     * These bytes can be ignored here. - To check if the alarm is in DSMR4 or SMR5 format, check
-     * the 9th byte (at index 8). If it is 0F, then it is SMR5, otherwise it is DSMR4, because the
-     * 9th byte in DSMR4 is in the identifier and this should be a number or a character, so it
-     * can't be ASCII code 0F.
+     * <p>Notes: - For MX382 and SMR5 alarms we get 8 addressing bytes in front of the alarm. These
+     * bytes can be ignored here. - To check if the alarm is in MX382, DSMR4 or SMR5 format, check
+     * the 9th byte (at index 8). If it is 0F, then it is SMR5, if it is C2 then it is MX382,
+     * otherwise it is DSMR4, because the 9th byte in DSMR4 is in the identifier and this should be
+     * a number or a character, so it can't be ASCII code 0F or C2.
```

Comment:
This long comment does not seem to be properly formatted (already before your changes). Are the HTML tags necessary? And some additional line breaks could make it more readable.

Reply:
I reformatted the comment with additional line breaks using HTML tags. These tags prevent automatically removing these line breaks